quicktravel_client.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'httparty', '0.15.7' # Bug in >16 relating to nested parameters
+  spec.add_dependency 'httparty', '~> 0.16'
```

Comment:
this should also specify a version greater than the one that fixes our issue eg `'~>0.16', '>=0.16.3424'` (I think thats the right syntax)

Reply:
Changing to '~> 0.17' as '0.17.1' is the version installed in Ecom while I do testing